react/components/SearchFilter.js

```diff
@@ -58,9 +58,15 @@ class SearchFilter extends Component {
       title = this.props.intl.formatMessage({ id: title })
```

Comment:
I know it probably wasn't you but, props aren't supposed to be changed like this, it's better to put another variable to have this condition and keep the prop unaltered.